compiler-rt/lib/asan/asan_fake_stack.cpp

```diff
@@ -311,7 +329,9 @@ extern "C" {
 // -asan-use-after-return=never, after modal UAR flag lands
 // (https://github.com/google/sanitizers/issues/1394)
 SANITIZER_INTERFACE_ATTRIBUTE
-void *__asan_get_current_fake_stack() { return GetFakeStackFast(); }
+void* __asan_get_current_fake_stack() {
```

Comment:
There is TODO for to remove this one
if this is just for test, can we instead rely on __asan_get_current_fake_stack
Use __builtin_frame_address

__builtin_frame_address always return real stack address

Reply:
Removed the argument and adjusted the test.

Reply:
Did you forget to upload the patch?

Reply:
I have rebased and updated it